### PR TITLE
fix to get release PR number

### PR DESCRIPTION
**Pull request by whoisxx**
## Proposed changes

- fix to get release PR number to compare CIRCIE_SHA1 variable

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -553,23 +553,3 @@ workflows:
           filters:
             branches:
               only: master
-
-  nightly-coverage:
```
Comment (aidan-kwon) [inline .circleci/config.yml:557]: Why did you remove nightly-coverage?

**Comment by aidan-kwon** [comment]
@whoisxx I think the description of this PR cannot explain the code change. 
Could you give more detailed information about the intention of this PR?

**Comment by whoisxx** [comment]
@aidan-kwon Sorry, there are other commits in it, I just update it.
`build_announce.sh` runs after upload RC binaries to S3 and it make comment to release PR that contained each binary links.
What I change is get PR number by matching `CIRCLE_SHA1` variable which is commit hash of RC.

File: .circleci/config.yml (hunk 1 above)
Comment (whoisxx) [reply .circleci/config.yml]: updated!
